hugo/themes/genome-portal-kit/archetypes/species/assembly/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Genome assembly"
+type: species_bundle
+layout: assembly
+lastmod: {{ .Date }}
```

Comment:
It could certainly suit to our needs, but perhaps providing the `lastmod` field everywhere leaves more flexibility to the (very hypothetical) user (related to the discussion above). But I have no strong opinion on this.